README.md

```diff
@@ -7,14 +7,45 @@ TypeScript types for the Seam API.
 
 ## Description
 
-TODO
+This repository aggregates the latest route schemas and type definitions for the Seam API.
+Upstream Seam repositories automatically contribute updates on every deployment
+via a Pull Request opened by Seam Bot.
+This ensures downstream consumers, e.g., the SDK, fakes, documentation generators, etc.,
+have a single source of truth that stays up to date with the API.
+
+### Structure
+
+The main entrypoint for this module is empty: there is dedicated entrypoint for each Seam API namespace.
+
+Each directory under `src/lib/seam/*` is owned by an upstream Seam repository.
+These files should not be edited manually as they will be overridden by automation.
+
+Each entrypoint may export one or more of the following:
```

Comment:
Should we export the openapi json (as a plain JS object)? @codetheweb

Reply:
Maybe better to point users at the hosted file? We could export `openapiUrl` 🤔

Reply:
yeah I think we should export it

personally I like having everything versioned under the same place